### PR TITLE
feature: aws lambda routing with tabs

### PR DESCRIPTION
**Pull request by tomek-ifm**

### CODE REVIEW

**Comment by dtuite** [comment]
@tomek-ifm can you squash these commits before we merge them please?

**Comment by tomek-ifm** [comment]
> @tomek-ifm can you squash these commits before we merge them please?

done

File: src/components/AWSLambdaPageTable/AWSLambdaPageTable.tsx
Comment (dtuite) [inline src/components/AWSLambdaPageTable/AWSLambdaPageTable.tsx:21]: No need to block this commit over it, but would you mind replacing `moment` with something else please? The project is in maintenance mode and there are far smaller libs out there which will do the job we need. e.g. [date-fns](https://date-fns.org/).

**Review by dtuite — APPROVED** [review]

**Comment by dtuite** [comment]
@tomek-ifm  I approve, but would like if either @iain-b or @mcalus3  can review before merge please.

File: src/components/AWSLambdaPageTable/AWSLambdaPageTable.tsx
Comment (mcalus3) [inline src/components/AWSLambdaPageTable/AWSLambdaPageTable.tsx:189]: I like to always wrap useContext with custom hook. That way you could have your custom logic there (for example error handling in case someone tries to use context outside of provider). But in this simple example it doesn't matter.

File: src/hooks/useServiceEntityAnnotations.ts
Comment (mcalus3) [inline src/hooks/useServiceEntityAnnotations.ts:30]: I just corrected that in firebase-functions plugin - since this now can't load of end with an error, just don't return "loading" and "errror" properties. It will simplify jsx in the component too.

**Review by iain-b — APPROVED** [review]

**Comment by dtuite** [comment]
@iain-b @tomek-ifm is this ready to merge? I see an approval 14 days ago?

I'm interested in getting this merged soon so the community can contribute to it.

**Comment by tomek-ifm** [comment]
> @iain-b @tomek-ifm is this ready to merge? I see an approval 14 days ago?
> 
> I'm interested in getting this merged soon so the community can contribute to it.

@dtuite I will merge it today, together with aws backend auth. Just wanted to add loader & change button to "show logs"

**Comment by tomek-ifm** [comment]
Closing in favor of https://github.com/RoadieHQ/backstage-plugin-aws-lambda/pull/2 which has overview widget instead of lambda functions table

**Pull request closed without merging**
